src/playerClass.ts

```diff
@@ -8,7 +8,7 @@ export class Player {
     if (name.length > 8) {
       throw new Error("Too long name");
     }
-    if (pokemons.length != 3) {
+    if (pokemons.length > 3) {
```

Comment:
I would prefer to do not change it and change implementation in a way that you create new Player() only when you have array of 3 pokemons, but I know it might be harder, so no pressure. Leave it like it is